src/issuance.rs

```diff
@@ -340,7 +340,7 @@ impl IssueBundle<Unauthorized> {
     /// * `WrongAssetDescSize`: If `asset_desc` is empty or longer than 512 bytes.
     pub fn add_recipient(
         &mut self,
-        asset_desc: String,
+        asset_desc: Vec<u8>,
```

Comment:
Let's go with `asset_desc: &Vec<u8>,` and copy/clone only when needed (line375)

Reply:
Done, with knock on changes